docs/Api/DefaultApi.md

```diff
@@ -73,51 +66,50 @@ $apiInstance = new onesignal\client\Api\DefaultApi(
     new GuzzleHttp\Client(),
     $config
 );
-$app_id = 'app_id_example'; // string | The OneSignal App ID for your app.  Available in Keys & IDs.
-$activity_id = 'activity_id_example'; // string | Live Activity record ID
-$begin_live_activity_request = new \onesignal\client\model\BeginLiveActivityRequest(); // \onesignal\client\model\BeginLiveActivityRequest
+$app_id = 'app_id_example'; // string
+$notification_id = 'notification_id_example'; // string
 
 try {
-    $apiInstance->beginLiveActivity($app_id, $activity_id, $begin_live_activity_request);
+    $result = $apiInstance->cancelNotification($app_id, $notification_id);
+    print_r($result);
 } catch (Exception $e) {
-    echo 'Exception when calling DefaultApi->beginLiveActivity: ', $e->getMessage(), PHP_EOL;
+    echo 'Exception when calling DefaultApi->cancelNotification: ', $e->getMessage(), PHP_EOL;
 }
 ```
 
 ### Parameters
 
 Name | Type | Description  | Notes
 ------------- | ------------- | ------------- | -------------
- **app_id** | **string**| The OneSignal App ID for your app.  Available in Keys &amp; IDs. |
- **activity_id** | **string**| Live Activity record ID |
- **begin_live_activity_request** | [**\onesignal\client\model\BeginLiveActivityRequest**](../Model/BeginLiveActivityRequest.md)|  |
+ **app_id** | **string**|  |
+ **notification_id** | **string**|  |
 
 ### Return type
 
-void (empty response body)
+[**\onesignal\client\model\GenericSuccessBoolResponse**](../Model/GenericSuccessBoolResponse.md)
 
 ### Authorization
 
-[app_key](../../README.md#app_key)
+[rest_api_key](../../README.md#rest_api_key)
 
 ### HTTP request headers
 
-- **Content-Type**: `application/json`
+- **Content-Type**: Not defined
```

Comment:
double check

Reply:
I think there is no change... the diffs are off, on the left shows `beginLiveActivity` method being deleted, and the right shows `cancelNotification` which didn't change and already had `**Content-Type**: Not defined`

Reply:
Ok that's what I figured. Just wanted to be sure 👍